jbrotli/src/main/java/org/meteogroup/jbrotli/io/BrotliInputStream.java

```diff
@@ -89,8 +89,9 @@ public synchronized void reset() throws IOException {
   private void readChunkFromInput() throws IOException {
     final byte[] uncompressedBuffer = new byte[INTERNAL_UNCOMPRESSED_BUFFER_SIZE];
     if (brotliStreamDeCompressor.needsMoreOutput()) {
-      final int length = brotliStreamDeCompressor.deCompress(new byte[0], uncompressedBuffer);
-      uncompressedInputStreamDelegate = new ByteArrayInputStream(uncompressedBuffer, 0, length);
+      final int[] lengths = brotliStreamDeCompressor.deCompress(new byte[0], uncompressedBuffer);
+      assert(lengths[0] == 0);
```

Comment:
Why this assertion?
AFAIK this will be removed by compiler when doing "production" compile (no debug info).